ocd/src/probe/flash/flasher.rs

```diff
@@ -290,6 +290,9 @@ impl<'a> Flasher<'a> {
         log::debug!("Halting core.");
         let cpu_info = flasher.target.core.halt(&mut flasher.probe);
```

Comment:
Is this still necessary? We reset and halt afterwards now anyways.

Reply:
I was able to remove this for M4. But when flashing to micro:bit the core came out sleeping after reset_and_halt. Might be an issue looking into.